scripts/cli-test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+# Runs a CLI-based integration test
+
+set -euxo pipefail
+export RUST_LOG=info
+export PEG_IN_AMOUNT=0.00099999
+source ./scripts/setup-tests.sh
+source ./scripts/build.sh
+source ./scripts/start-fed.sh
+source ./scripts/start-gateway.sh
+source ./scripts/pegin.sh
+
+#### BEGIN TESTS ####
+
+# reissue
+TOKENS=$($MINT_CLIENT spend 42000)
+$MINT_CLIENT reissue $TOKENS
+$MINT_CLIENT fetch
+
+# peg out
+PEG_OUT_ADDR="$($BTC_CLIENT getnewaddress)"
+$MINT_CLIENT peg-out $PEG_OUT_ADDR 500
+sleep 5 # wait for tx to be included
+mine_blocks 120
+await_block_sync
+sleep 15
+mine_blocks 10
```

Comment:
i think we could actually replace that sleep with an `until` loop that watches for any unconfirmed tx to a certain address (or any tx being submitted for that matter).

*(I'm aware that I wrote that shitty code in the first place, just reconsidering)*

Reply:
Yep I can fix that.